### PR TITLE
Init: add ensure type

### PR DESCRIPTION
**Pull request by danirabbit**
Does the least to give a way for Adwaita to detect that Granite has been initialized so that it can disable styles i.e. for Epiphany

### CODE REVIEW

File: lib/Init.vala [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
 namespace Granite {
-    static bool initialized = false;
+    private static bool initialized = false;
 
-    internal void init () {
+    /*
+     * Initializes Granite.
+     * If Granite has already been initialized, the function will return.
+     * Makes sure translations and types for Granite are set up properly.
+     */
```
Comment (Exalm) [inline lib/Init.vala:13]: Missing `Since:`?

File: lib/Init.vala [hunk 2]
```diff
@@ -4,13 +4,20 @@
...
+    public void init () {
         if (initialized) {
             return;
         }
 
+        typeof (Granite.Settings).ensure ();
```
Comment (Exalm) [inline lib/Init.vala:19]: You're already inside Granite namespace, so jut
```suggestion
        typeof (Settings).ensure ();
```
would have worked. Might be a matter of code style tho, not sure.

**Review by Exalm — APPROVED** [review]
LGTM otherwise

File: lib/Init.vala (hunk 2 above)
Comment (danirabbit) [reply lib/Init.vala:19]: Yeah I just want to make sure we disambiguate between GLib.Settings just in case!

**Comment by Exalm** [comment]
Works fine on libadwaita side.

**Review by tintou — APPROVED** [review]
One might revisit this init function to use a GLib.Once but that's just a nit

**Pull request merged**